### PR TITLE
Update untracked script

### PR DESCRIPTION
**Pull request by flodolo**
List also folders for locales completely unknown.

### CODE REVIEW

**Comment by flodolo** [comment]
@TheoChevalier r?

I still get one file on website 12 (Arabic)

File: app/scripts/untracked_files [hunk 1]
```diff
@@ -45,7 +45,7 @@ $ignored_files = [
     'privacy/notices/firefox-hello.lang',
     'privacy/notices/firefox.lang',
 ];
-$ignored_locales = ['ja'];
+$ignored_locales = ['hi', 'ja'];
```
Comment (TheoChevalier) [inline app/scripts/untracked_files]: Why are we ignoring hi? There's probably a reason, but it's not obvious to me :)

Comment (flodolo) [inline app/scripts/untracked_files]: We don't support `hi`, we support `hi-IN`. But there's an activation file for legal-docs for the former.
https://github.com/mozilla-l10n/www.mozilla.org/tree/master/hi

**Comment by TheoChevalier** [comment]
./untracked_files 0 returns a file for 'fur' locale, but we can't display untracked files for unsupported locales: ./untracked_files 0 fur returns Unsupported locale #fur.

Maybe we should change that?

**Comment by TheoChevalier** [comment]
lgtm otherwise :snail:

**Comment by flodolo** [comment]
Try updating www.mozilla.org? I [removed](https://github.com/mozilla-l10n/www.mozilla.org/commit/2f323617fa672291f18368066c43bba12046538a) the file about an hours ago, it's a remnant from PHP and it didn't make much sense to keep it

**Comment by TheoChevalier** [comment]
My point was more about the different info returned regarding the different arguments, rather than the file itself :)

**Comment by flodolo** [comment]
Well, I think it's by design: without specifying a locale we check the entire website, so you might get "unsupported" locales.

**Pull request merged**